local/bin/py/build/actions/integrations.py

```diff
@@ -864,6 +866,22 @@ def process_integration_readme(self, file_name, marketplace=False, content=None)
             #         print(f"removing {integration_name} due to is_public/display_on_public_websites flag, {out_name}")
             #         remove(out_name)
 
+    def retrieve_page_description(self, page_url):
+        page = requests.get(page_url, timeout=1)
+        page_content = bs(page.content, 'html.parser')
+        metas = page_content.find_all('meta')
+        page_description = ''
+        if metas:
+            for meta in metas:
+                if 'property' in meta.attrs and meta.attrs['property'] == 'og:description':
+                    page_description = fr"{meta.attrs['content']}"
+        if not page_description:
+            page_description = fr"{page_content.title.string}"
+            page_description = page_description.replace(" | Datadog", "")
```

Comment:
I think you might be able to query the meta property directly:
```suggestion
        page_description = None
        og_description = page_content.find('meta', property='og:description')
        if og_description:
            page_description = og_description.get('content', fr"{page_content.title.string}")
            page_description = page_description.replace(" | Datadog", "")
```